entrez-direct/7.50.20171103/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Base Image
+FROM biocontainers/biocontainers:latest
+
+# Metadata
+LABEL base.image="biocontainers:latest"
+LABEL version="3"
+LABEL software="entrez-direct"
+LABEL software.version="7.50.20171103"
+LABEL description="Entrez Direct (EDirect) is an advanced method for accessing the NCBI's set of interconnected Entrez databases"
+LABEL website="https://github.com/ncbi/magicblast"
+LABEL documentation="https://ncbi.github.io/magicblast/"
+LABEL License_file = /home/biodocker/bin/LICENSE
+LABEL tags="BLAST"
```

Comment:
would be nice to ad license in addition to license_file to know what is the license type